src/index.ts

```diff
@@ -182,15 +188,20 @@ const plugin: PluginImpl<Options> = (options = {}) => {
       const source = ts.createSourceFile(chunk.fileName, code, ts.ScriptTarget.Latest, true);
       const fixer = new NamespaceFixer(source);
 
-      code = Array.from(typeReferences, ref => `/// <reference types="${ref}" />`).join("\n");
-      if (code) {
-        code += "\n";
-      }
+      code = writeBlock(Array.from(typeReferences, ref => `/// <reference types="${ref}" />`));
+      code += writeBlock(Array.from(moduleDeclarations, moduleDeclaration => moduleDeclaration));
       code += fixer.fix();
 
       return { code, map: { mappings: "" } };
     },
   };
 };
 
+function writeBlock(codes: string[]): string {
```

Comment:
```suggestion
function writeBlock(codes: Array<string>): string {
```
just personal preference ;-)